src/frontend/next/src/pages/LogoIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from 'react';
+import Image from 'next/image';
```

Comment:
```
7:25:05 PM: Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
Possible solutions:
7:25:05 PM:   - Use `next start`, which starts the Image Optimization API.
7:25:05 PM:   - Use Vercel to deploy, which supports Image Optimization.
7:25:05 PM:   - Configure a third-party loader in `next.config.js`.
7:25:05 PM: Read more: https://err.sh/next.js/export-image-api
7:25:05 PM:     at exportApp (/opt/build/repo/src/frontend/next/node_modules/next/dist/export/index.js:14:712)
```

We can't use `Image` unfortunately, it only works with server-side optimization (ie., you have to run a server to create dynamic images on demand).  For pure static/exported projects, it's not possible.

Reply:
That's too bad. Ah well, I removed it.